.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+name: "CI"
+on: [push, pull_request]
```

`pull_request` means only once per PR, whereas `push` means once per commit
```suggestion
on: push
```